src/viam/media/video.py

```diff
@@ -18,33 +28,40 @@ class RawImage(NamedTuple):
     mime_type: str
     """The mimetype of the image"""
 
-    width: int
-    """The width of the image"""
-
-    height: int
-    """The height of the image"""
-
     def close(self):
         """Close the image and release resources. For RawImage, this is a noop."""
         return
 
 
 class CameraMimeType(str, Enum):
-    RAW = "image/vnd.viam.rgba"
+    VIAM_RGBA = "image/vnd.viam.rgba"
     JPEG = "image/jpeg"
     PNG = "image/png"
     PCD = "pointcloud/pcd"
+    UNSUPPORTED = "unsupported"
+
+    @classmethod
+    def from_lazy(cls, value: str) -> Tuple[Self, bool]:
+        is_lazy = False
+        mime_type = value
+        if value.endswith(LAZY_SUFFIX):
+            mime_type = value[: (len(value) - len(LAZY_SUFFIX))]
+            is_lazy = True
+        if not cls.is_supported(value) and not is_lazy:
+            mime_type = CameraMimeType(CameraMimeType.UNSUPPORTED)
+        return (cls(mime_type), is_lazy)
+
+    @property
+    def with_lazy_suffix(self) -> str:
+        return f"{self.value}{LAZY_SUFFIX}"
 
     def encode_image(self, image: Union[Image, RawImage]) -> bytes:
         if isinstance(image, RawImage):
             return image.data
-        if self == CameraMimeType.JPEG:
-            buf = BytesIO()
-            image.save(buf, format="JPEG")
-            return buf.getvalue()
-        if self == CameraMimeType.PNG:
+
+        if self.name in LIBRARY_SUPPORTED_FORMATS:
```

Comment:
[q] there is no chance that self.name will have `lazy` appended to it?

Reply:
Similarly to above: nope! Name can only be as we define it, so as long as we don't define a lazy member, it can't happen